src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Interop/BrowserInteropHelper.cs

```diff
@@ -41,53 +32,13 @@ static BrowserInteropHelper()
         /// <summary>
         /// Returns true if the app is a browser hosted app.
         /// </summary>
-        /// <remarks>
```

Comment:
Was this intentionally removed?

Reply:
Yeah, `HostingFlags` was an internal static property that is no longer present since #9855, just forgot to clean up before.

Reply:
Yeah makes sense. Other changes seems fine. Will speed up the test pass and try to get this in.